librz/core/cmd_debug.c

```diff
@@ -4425,24 +4426,26 @@ static int cmd_debug_continue(RzCore *core, const char *input) {
 		return 1;
 	}
 	case 'u': // "dcu"
-		if (input[2] == '?') {
+		if (input[1] == '?') {
 			rz_core_cmd_help(core, help_msg_dcu);
-		} else if (input[2] == '.') {
+		} else if (input[1] == '.') {
 			cmd_dcu(core, "cu $$");
 		} else {
-			cmd_dcu(core, input);
+			char *tmpinp = rz_str_newf("cu %s", input + 2);
```

Comment:
Ok the last errors are probably here.. I think I messed up a bit with the call to `cmd_dcu`. Could you check what `cmd_dcu` expects (in the old code) and update this accordingly?